### ISSUE TITLE
[FE] Ensure page loading optimized for screen readers (pre-check-in)

### ISSUE THREAD

**Issue by niradatta1** [department-of-veterans-affairs/va.gov-team#35471]
Based on user testing, we found that voice overs are unable to tell if a page is loading when transitioning between pages. It's recommended we provide some text for screen readers (e.g. "page is loading") when a user transitions between pages.

**Comment by ytsaoca** [comment]
Add loading after logging in or submitting the for
![Verify Identity](https://user-images.githubusercontent.com/90633685/151199072-810e2432-0f5f-4d74-b61d-39ba1422a4ad.jpg)
m

**Comment by brianseek** [comment]
Hi @joshkimux - we are trying to determine the best solution for this issue. One thought was to add an ARIA alert or maybe ARIA status to the loading component, but not sure if this is an overuse of alerts. Do you have an ideas on the best way to achieve this? Thanks

**Comment by joshkimux** [comment]
Adding `role="status"` to the container that holds the "loading" message would be preferred here, especially if there's a visible message. It's automatically set to be polite (instead of assertive which is implicitly held by `role="alert"` which will yell at folks) ✌️ 

More from WCAG: [Using role=status to present status messages](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22.html)

**Comment by brianseek** [comment]
thanks @joshkimux !

**Comment by brianseek** [comment]
@ytsaoca & @niradatta1 This fix is ready for validation, but lets hold on it until the other screen reader fixes are merged and we can review as a group.
